### PR TITLE
🎨 Feature/limit UI update

### PR DESCRIPTION
**Pull request by alfetopito**
# Summary

Big re-iteration of limit orders

Applying the latest designs from [figma](https://www.figma.com/file/9UJGrVP7AYtSa3M35uQPK5/2022---Limit-orders?node-id=1627-5286&t=eElItvKtmLzbVfag-0)

# Features

## High fee warning
image

In the limit orders form, it now shows an explanation why the sell amount is too small, comparing with current estimated fees

## Refreshed open orders table
image

### Market price column
Added `Market price` column, with the current price you could see on Coingecko for example
This is not volume based. AKA Spot price.

### Executes at column
Added `Executes at` column, with the current market conditions taken into account.
It'll consider the current fee and the volume based market price to show what is the probable execution price.
It has colored indicators to reflect how close the order is to execution
It will show `unfillable` when we estimate the order cannot be executed

On hover, it'll show details on how close/far the estimated price is from current spot price:
image

Similar to the `High fee warning` on the limit orders form, there will also be a warning when the current estimated fee is higher than a percentage of your sell amount:
image

# Background

Base feature branch for limit orders ui update
Supersedes https://github.com/cowprotocol/cowswap/pull/2091

Merged with latest `develop`

### CODE REVIEW

**Comment by elena-zh** [comment]
Hey @fairlighteth , in this PR I see all issues I reported in https://github.com/cowprotocol/cowswap/pull/2091#issuecomment-1462646079 , and no fixes for them. Could you please check whether they were deployed here? 

Thank you!

**Comment by elena-zh** [comment]
Hey @alfetopito , duplicating my comment from https://github.com/cowprotocol/cowswap/pull/2135#issuecomment-1465180421: 

But there is 1 nitpick: when place 2 orders (WETH/GNO, then another with GNO/WETH), and press on the revert button, price in the 1st order  will not be reverted. See the video: https://www.loom.com/share/cbd4de7aeb9a4b358e0d77d6db0c0529

Thanks!

**Comment by elena-zh** [comment]
Duplicating issues from the previous PR here for traceability: issues reported in https://github.com/cowprotocol/cowswap/pull/2108#issuecomment-1462628821 are not fixed and reproducible here.

File: .eslintrc.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "no-unused-vars": "off",
     "unused-imports/no-unused-imports": "error",
     "unused-imports/no-unused-vars": [
-      "warn",
+      "error",
```
Comment (alfetopito) [inline .eslintrc.json:32]: Be warned, unused vars is now an error!!

@cowprotocol/frontend

File: src/cow-react/modules/limitOrders/utils/calculateOrderExecutionStatus.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  if (!difference) {
+    return undefined
+  }
+
+  console.log(
+    'diff',
+    difference.toFixed(2),
+    LOWER_PERCENTAGE_DIFFERENCE.toFixed(2),
+    UPPER_PERCENTAGE_DIFFERENCE.toFixed(2)
+  )
```
Comment (alfetopito) [inline src/cow-react/modules/limitOrders/utils/calculateOrderExecutionStatus.ts:18]:
```suggestion
```

**Pull request merged**